tests/plugins/uploadwidget/manual/destroyeditor.html

```diff
@@ -0,0 +1,21 @@
+<div id="editor"></div>
+<button id="destroyer">Destroy</button>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		imageUploadUrl: 'http://sub.ckeditor.dev/',
+		on: {
+				instanceReady: function( evt ) {
+					var editor = evt.editor;
+
+					document.getElementById( 'destroyer' ).addEventListener( 'click', function() {
```

Comment:
IE 8 does not support `addEventListener`, which results in error in this browser and not ignoring the test.